rules/Proj2005.md

```diff
@@ -0,0 +1,30 @@
+# Proj2005: Escape XML nodes of resource values
+In some scenario's is can be useful to have some XML (HTML) based markup in
+resources files. It is strongly recommanded to escape that XML using either
+`<![CDATA[`/`]]>` or `&lt;`/`@gt;` escaping sequences.
```

Comment:
```suggestion
In some scenarios it can be useful to have some XML (HTML) based markup in
resources files. It is strongly recommended to escape that XML using either
`<![CDATA[`/`]]>` or `&lt;`/`@gt;` escaping sequences.
```